demo/package.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "demo",
   "scripts": {
-    "start": "webpack serve --mode development",
-    "build": "webpack --mode production",
+    "start": "parcel index.html",
+    "build": "parcel build index.html",
     "publish": "npm run build && gh-pages -d dist",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prestart": "npm link .."
+  },
+  "dependencies": {
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0"
```

Comment:
I know this is a debatable point but I prefer build-time dependencies to be considered as `devDependencies`.